content-src/components/DebugPage/DebugPage.js

```diff
@@ -63,20 +62,16 @@ const DebugPage = React.createClass({
             <div className="form-group">
               <label>Data Source</label>
               <select value={this.state.dataSource} onChange={e => this.setState({dataSource: e.target.value})}>
-                {Object.keys(this.props.raw).map(source => {
-                  return (<option key={source} value={source}>{source}</option>);
-                })}
+                {Object.keys(this.props.raw).map(source => (<option key={source} value={source}>{source}</option>))}
               </select>
             </div>
           </div>
           <div>
             {this.state.component === "Spotlight" &&
               <div className="spotlight">
-                {selectSpotlight({
-                  Highlights: this.props.raw[this.state.dataSource]
-                }).rows.map((item, i) => {
-                  return (<SpotlightItem key={i} {...item} />);
-                })}
+                {selectSpotlight({Highlights: this.props.raw[this.state.dataSource]}).rows.map((item, i) =>
+                  (<SpotlightItem key={i} {...item} />))
+                }
```

Comment:
This is partially `object-curly-newline` as well as http://eslint.org/docs/rules/arrow-body-style where an arrow function returning a value doesn't need the curly and `return`.